src/disasm.cpp

```diff
@@ -886,542 +885,542 @@ const BYTE CDetourDis::s_rbModRm[256] = {
 
 const CDetourDis::COPYENTRY CDetourDis::s_rceCopyTable[257] =
```

Comment:
How about we remove this explicit 257, remove the terminal entry, and C_ASSERT that the size is 256? That gives some of the sanity check, w/o requiring the tedious recording of the numbers, aside from comments which have less stringent requirement.